srcpkgs/xremap/template

```diff
@@ -0,0 +1,27 @@
+# Template file for 'xremap'
+pkgname=xremap
+version=0.4.4
+revision=1
+build_style=cargo
+short_desc="Dynamic key remapper for X11 and Wayland"
+maintainer="akhil <akhil.j@outlook.in>"
+license="MIT"
+homepage="https://github.com/k0kubun/xremap"
+changelog="https://raw.githubusercontent.com/k0kubun/xremap/master/CHANGELOG.md"
+distfiles="https://github.com/k0kubun/xremap/archive/refs/tags/v${version}.tar.gz"
+checksum=323cc9a900e7bc226008179619e49ee679caa6c9ade85e69bb5e710ffb0ed7ff
+
+post_build() {
+	xremap=$(find $GOPATH/bin -name xremap)
+	for shell in bash elvish fish powershell zsh; do
```

Comment:
We can only really install bash, fish and zsh completions

Reply:
why?  
xremaps can generate all. 
```
        --completions <SHELL>    Generate shell completions [possible values: bash, elvish, fish,
                                 powershell, zsh]
```

Reply:
oops. powershell is for windows. i got it.

Reply:
`vcompletion` only accepts those three, see in the manual